chia/cmds/wallet_funcs.py

```diff
@@ -33,6 +34,37 @@ def print_transaction(tx: TransactionRecord, verbose: bool, name) -> None:
         print("")
 
 
+def get_csv_line(tx: TransactionRecord, name, date_fmt) -> None:
+
+    chia_amount = Decimal(int(tx.amount)) / units["chia"]
+    fee_amount = Decimal(int(tx.fee_amount)) / units["chia"]
+    to_address = encode_puzzle_hash(tx.to_puzzle_hash, name)
+    txType = get_tx_type(tx.type);
+    tx_datetime = datetime.fromtimestamp(tx.created_at_time).strftime(date_fmt)
+    status = 'Confirmed' if tx.confirmed else ('In mempool' if tx.is_in_mempool() else 'Pending')
+
+    csv_line = ""
+    csv_line += f"{tx.wallet_id},"
+    csv_line += f"{tx_datetime},"
+    csv_line += f"{txType},"
+    csv_line += f"{status},"
+    csv_line += f"{tx.confirmed_at_height},"
+    csv_line += f"{tx.name},"
+    csv_line += f"{chia_amount},"
+    csv_line += f"{fee_amount},"
+    csv_line += f"{to_address}"
+
+    return(csv_line)
```

Comment:
Hello, a few unsolicited notes that could reduce code verbosity. This block looks like
```python
line_elems = [
    tx.wallet_id,
    tx_datetime, 
    # etc.
    to_address,
]
DELIM = ","
return DELIM.join(map(str, line_elems))